crates/rspack_plugin_runtime/src/array_push_callback_chunk_format.rs

```diff
@@ -122,14 +128,20 @@ async fn render_chunk(
     }
     source.add(RawStringSource::from_static(")"));
   } else {
-    let chunk_loading_global = &compilation.options.output.chunk_loading_global;
+    let chunk_loading_global = compilation
+      .options
+      .output
+      .chunk_loading_global
+      .render(PathData::default().runtime(&runtime_chunk_runtime_id(chunk, compilation)))
+      .await?;
+    let chunk_loading_global = json_stringify(&chunk_loading_global);
 
     source.add(RawStringSource::from(format!(
-      r#"({}["{}"] = {}["{}"] || []).push([[{}], "#,
+      r#"({}[{}] = {}[{}] || []).push([[{}], "#,
```

Comment:
This is the hard part, runtime chunk is easy to adapt, but for normal chunk, they can have multiple runtime, we need to call `chunkLoadingGlobal_[runtime]` multiple times to "push" the chunk into every runtime